### DISCUSSION TITLE
Does Fastlane affect your existing apps in the stores?

### DISCUSSION THREAD

**Discussion by kirakoki** [fastlane/fastlane#22059] [General]
We are currently preparing to launch a new app using Fastlane, and have successfully implemented numerous automations including the creation of certificates, provisioning profiles, p12 files, and automating the deployment process to both the App Store and Google Play etc.

At present, we have an existing app live on both the Apple and Google Play stores. As we proceed with Fastlane for our new app, we want to ensure that this new push remains completely separate, with distinct distribution and notification certificates.

Could anybody please confirm whether running Fastlane again for this new app will in any way affect our existing app or its certificates? Specifically, will Fastlane overwrite or alter the existing certificates and provisioning profiles or have a risk to break existing setups in any way?

**Comment by julie598doyle** [comment]
Hello,
Running Fastlane for your new app will not [Snaptik](https://www-snaptik.com) affect your existing app or its certificates, as long as you maintain separate configurations, unique app identifiers, and distinct certificates and provisioning profiles. Ensure each app has its own Fastlane directory and configuration files, and use unique bundle identifiers for iOS and application IDs for Android.

Best Regards,
Julie Doyle
